app/services/decisions/outgoings_decision_tree.rb

```diff
@@ -29,5 +29,25 @@ def after_housing_payment_type
         edit(:council_tax)
       end
     end
+
+    def after_outgoings_more_than_income
+      if requires_full_capital
+        edit('/steps/capital/property_type')
+      else
+        edit('/steps/capital/trust_fund')
+      end
+    end
+
+    def requires_full_capital
```

Comment:
I wonder if its time to start thinking about moving this sort of general logic out of the individual decision trees. Perhaps like we've done with the TypeOfApplication module?